### ISSUE TITLE
Automatically update tool config file (e.g Flow)

### ISSUE THREAD

**Issue by radex** [greenkeeperio/greenkeeper#989]
Every time Greenkeeper updates `flow-bin` dependency, the build will break, because `.flowconfig` needs to be updated too to the same version for CI to pass. I'm sure there's many other tools as well that need another file to be updated to work correctly.

**I'm willing to implement this myself**, just let me know if such a change would be accepted, and maybe some basic pointers to where I'd need to change code to get this done.

**Comment by espy** [comment]
Hi Radex, thanks for getting in touch.
Never having used flow before, is this about the `version` key in the `.flowconfig` getting updated to the new version of `flow-bin`, or does it need to be updated in a more complex manner?

**Comment by radex** [comment]
@espy Exactly, `version = ^x.x.x` need to be updated if flow-bin dependency is updated.
